neurokit2/hrv/hrv_frequency.py

```diff
@@ -265,12 +265,19 @@ def _hrv_frequency_show(
         __, ax = plt.subplots()
 
     frequency_band = [ulf, vlf, lf, hf, vhf]
+
+    # Compute sampling rate for plot windows
+    if sampling_rate is None:
+        med_sampling_rate = np.median(np.diff(t))
```

Comment:
Can we add a comment here to justify this decision, e.g.

```suggestion
        med_sampling_rate = np.median(np.diff(t))  # This is just for visualization purposes (see #800)
```

Reply:
OK thanks, I am sure you'll be correct! I don't have the code open right now!

Reply:
I've now added a separate comment to `signal_psd()`, thank you :)